### DISCUSSION TITLE
how count parameters number on spacy models

### DISCUSSION THREAD

**Discussion by info2000** [explosion/spaCy#9730] [Help: Other Questions] [answered]
How can I know the data to count the model parameters?
I want to know how many million parameters is trained my models, like explained here
https://towardsdatascience.com/counting-no-of-parameters-in-deep-learning-models-by-hand-8f1716241889

But I don't know how get the input, output and hidden layer size

Thanks

**Comment by adrianeboyd** [comment, marked as answer]
Hi, the closest tool we have related to this is `spacy debug model`: https://spacy.io/api/cli#debug-model. You can take a look at the sample outputs in the expandable box just above the list of parameters.

**Reply by info2000** [reply]
Awesome, thanks
